### ISSUE TITLE
show topics; Error reading field 'version':

### ISSUE THREAD

**Issue by wangshao1** [confluentinc/ksql#1014]
```
[{
"error":{"statementText":"show topics ; ",
"errorMessage":{
  "message":"Error reading field 'version': java.nio.BufferUnderflowException",
  "stackTrace":["org.apache.kafka.common.protocol.types.Schema.read(Schema.java:76)","org.apache.kafka.clients.consumer.internals.ConsumerProtocol.deserializeAssignment(ConsumerProtocol.java:105)","kafka.admin.AdminClient$$anonfun$12.apply(AdminClient.scala:329)","kafka.admin.AdminClient
```

**Comment by wangshao1** [comment]
kafka 1.0
ksql 0.4 and 0.5 
too many tpics?

**Comment by apurvam** [comment]
Can you post the relevant section of the ksql.log file?

**Comment by fengzheng1024** [comment]
there it is.
1. 
[ksql.log](https://github.com/confluentinc/ksql/files/1836331/ksql.log)

**Comment by apurvam** [comment]
Hmm. that is strange, it can't deserialize the response of the ConsumerGroupCommand

```
[2018-03-22 10:44:13,510] ERROR Failed to handle POST:io.confluent.ksql.rest.entity.KsqlRequest@89ddc523 (io.confluent.ksql.rest.server.resources.KsqlResource:151)
org.apache.kafka.common.protocol.types.SchemaException: Error reading field 'version': java.nio.BufferUnderflowException
	at org.apache.kafka.common.protocol.types.Schema.read(Schema.java:76)
	at org.apache.kafka.clients.consumer.internals.ConsumerProtocol.deserializeAssignment(ConsumerProtocol.java:105)
	at kafka.admin.AdminClient$$anonfun$12.apply(AdminClient.scala:329)
	at kafka.admin.AdminClient$$anonfun$12.apply(AdminClient.scala:326)
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:234)
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:234)
	at scala.collection.Iterator$class.foreach(Iterator.scala:891)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1334)
	at scala.collection.IterableLike$class.foreach(IterableLike.scala:72)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:54)
	at scala.collection.TraversableLike$class.map(TraversableLike.scala:234)
	at scala.collection.AbstractTraversable.map(Traversable.scala:104)
	at kafka.admin.AdminClient.describeConsumerGroup(AdminClient.scala:326)
	at io.confluent.ksql.util.KafkaConsumerGroupClientImpl.describeConsumerGroup(KafkaConsumerGroupClientImpl.java:77)
```

This is a kafka tool. Can you enable TRACE logging for the `kafka.admin` package in your log4j file? 

Basically add the following `log4j.logger.kafka.admin=DEBUG, kafka` to your `config/log4j-rolling.properties` file, restart the ksql server, and run `show topics` again. Then share the ksql-kafka.log file. 

Thanks!

**Comment by fengzheng1024** [comment]
Do you mean this?

# this is a sample log4j config that will roll log files
# lines with `File=` may need to be updated for your environment

log4j.rootLogger=INFO, main
log4j.logger.kafka.admin=DEBUG,kafka
# appenders
log4j.appender.main=org.apache.log4j.RollingFileAppender


but sql-kafka.log not change.

**Comment by miguno** [comment]
@douglarek:

> any updates ? I am using KSQL 5.0.0-beta1, and still have this issue

Which version of Kafka (brokers) are you using?

**Comment by douglarek** [comment]
> Which version of Kafka (brokers) are you using?

kafka 1.0.1

**Comment by agavra** [comment]
It seems that this has been resolved in later versions of Kafka 1.1.1+. In order to keep our backlog tidy, I'm closing out this issue but feel free to re-open or open a new ticket if the problem persists.
